### PR TITLE
when the data is int dtype,fixed the issue of roll crash

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

**Comment by liangs6212** [comment]
issue link: #4438

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/ZOO-PR-Validation-Spark-3.1/1703/
http://10.112.231.51:18888/job/ZOO-PR-Validation/6262/

**Comment by TheaperDeng** [comment]
we will cast all the output to np.float32

File: pyzoo/test/zoo/chronos/data/test_tsdataset.py [hunk 1]
```diff
@@ -300,13 +309,24 @@ def test_tsdataset_roll_order(self):
                                        id_col="id")
         x, y = tsdata.roll(lookback=1, horizon=1, id_sensitive=False).to_numpy()
         assert x.shape == (2, 1, 3) and y.shape == (2, 1, 1)
-        assert np.array_equal(x, np.array([[[1.9, 1, 2]], [[2.3, 0, 9]]]))
-        assert np.array_equal(y, np.array([[[2.4]], [[2.6]]]))
+        assert np.array_equal(x, np.array([[[1.9, 1, 2]], [[2.3, 0, 9]]], dtype=np.float32))
```
Comment (TheaperDeng) [inline pyzoo/test/zoo/chronos/data/test_tsdataset.py:312]: do we still need this dtype param?

**Review by TheaperDeng — APPROVED** [review]
pls run jenkins

File: pyzoo/test/zoo/chronos/data/test_tsdataset.py (hunk 1 above)
Comment (liangs6212) [reply pyzoo/test/zoo/chronos/data/test_tsdataset.py:312]: yep, np.array.dtype is float64.

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/ZOO-PR-Python-ExampleTests-py36-ray/2597/
http://10.112.231.51:18888/job/ZOO-PR-Validation/6270/

**Pull request merged**